install/installer/pkg/components/server/types.go

```diff
@@ -107,6 +107,7 @@ type GitHubApp struct {
 	CertPath        string `json:"certPath"`
 	MarketplaceName string `json:"marketplaceName"`
 	LogLevel        string `json:"logLevel"`
+	CertSecretname  string `json:"certSecretName"`
```

Comment:
small nit: `CertSecretName` (upper case `Name`) would fit with JSON and other `Name`s